language/move-ir-compiler/transactional-tests/tests/signer/signer_keyword.mvir

```diff
@@ -0,0 +1,7 @@
+//# print-bytecode
+module 0x0.M {
```

Comment:
FWIW, in #9941 I'm working on allowing these tests to remain scripts and still have their bytecode printed out. Until then, this is the way to go, thanks!